plugins/jdg62/pom.xml

```diff
@@ -75,19 +75,6 @@
          </exclusions>
       </dependency>
 
-      <dependency>
```

Comment:
This pom.xml doesn't use infinispan-bom. I would like to keep as yes.

Reply:
I only removed `narayana-jta` because is not being used, WDYT?